### DISCUSSION TITLE
Pointer constraints

### DISCUSSION THREAD

**Discussion by avsej** [flxzt/rnote#765] [Ideas]
I wonder if it would be possible to constraint the pointer to the canvas, so that the pen won't be able to touch any of the controls accidentally. I see that wayland has API for this: https://smithay.github.io/smithay/wayland_protocols/wp/pointer_constraints/index.html

Or maybe in some other way. Something like what virtualbox does with the pointer when it grabs the mouse control.

**Comment by dehesselle** [comment]
Are you aware that v0.7.0 introduced a "focus mode" to disable/remove all the controls? Doesn't that solve your problem?

**Comment by avsej** [comment]
Yes, it solves the problem. Thank you
